job-utilities/local_development/local_scheduler.py

```diff
@@ -46,22 +62,39 @@ def create_schedule():
             if job_details["scheduling"]["type"] == "cron":
                 hours = str(job_details["scheduling"]["timing"]["hours"]).zfill(2)
                 minutes = str(job_details["scheduling"]["timing"]["minutes"]).zfill(2)
-                print("Scheduling job " + job_name + " at " + hours + ":" + minutes)
+                logger.info("Scheduling job " + job_name + " at " + hours + ":" + minutes)
 
                 schedule.every().day.at(hours + ":" + minutes).do(run_job,
                                                       job_details=job_details, job_name=job_name)
             elif job_details["scheduling"]["type"] == "interval":
                 minutes = job_details["scheduling"]["timing"].get("minutes", 0)
                 hours = job_details["scheduling"]["timing"].get("hours", 0)
                 total_minutes = minutes + hours*60
-                print("Scheduling job for every " + str(total_minutes) + " minutes")
+                logger.info("Scheduling job for every %d minutes", total_minutes)
 
                 schedule.every(total_minutes).seconds.do(run_job, job_details=job_details,
                                                          job_name=job_name)
 
-if __name__ == '__main__':
-    create_schedule()
+def main():
+    """ The primary interface for this script. """
+    parser = argparse.ArgumentParser(description="External CMR scheduler")
+    parser.add_argument('-t', '--test', action='store_true',
+                        help='Do a test run of RefreashKMSCache and exit.')
+    args = parser.parse_args()
 
+    if args.test:
```

Comment:
I needed a quick and dirty way to run the code and verify that urllib3 was functioning on the new version.